### PR TITLE
Make the title a link to the prop

### PR DESCRIPTION
**Pull request by adrianmcli**
This is for when a proposal has been selected (i.e. `selectedProposal === true`). It would be nice for the user to just click the big heading and have the proposal open up.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -67,9 +67,16 @@ export default function Home({ openProposals, fallback }: HomePageProps) {
         </div>
         <div className="md:w-2/3 md:ml-auto">
           <h1 className="text-3xl font-semibold  m-4 px-2 pt-2">
-            {selectedProposal
-              ? `${selectedProposal.id}: ${selectedProposal.title}`
-              : 'Vote Timeline'}
+            {selectedProposal ? (
+              <a
+                href={`https://nouns.wtf/vote/${selectedProposal.id}`}
+                target="_blank"
```
Comment (wilsoncusack) [inline pages/index.tsx:73]: add `rel="noreferrer"` and probably className hover:underline, also can use `getNounsLink` helper func 

be sure to run yarn lint:fix locally before pushing and fixing any errors (warnings OK)

**Review by wilsoncusack — COMMENTED** [review]
Thanks for this @adrianmcli! Build failed due to missing noreferrer, see my comment for some other notes

File: pages/index.tsx (hunk 1 above)
Comment (adrianmcli) [reply pages/index.tsx:73]: gotcha! Lemme fix that for ya.

**Comment by adrianmcli** [comment]
looks like I'll need authorization to see if it deploys properly

File: pages/index.tsx (hunk 1 above)
Comment (wilsoncusack) [reply pages/index.tsx:73]: looks like we missed using [getNounsLink](https://github.com/wilsoncusack/nouns-vote-with-reason/blob/d0ff589ffcaab1884c6ccaec7d4cc754024baa35/lib/util/link.ts#L5), but it works :)

Comment (adrianmcli) [reply pages/index.tsx:73]: ah sorry, I missed that.

Comment (adrianmcli) [reply pages/index.tsx:74]: added!

**Comment by wilsoncusack** [comment]
oof sorry @adrianmcli just got in #16 and looks like we have a conflict :(

**Comment by wilsoncusack** [comment]
oops meant https://github.com/wilsoncusack/nouns-vote-with-reason/pull/20

**Comment by adrianmcli** [comment]
yay, thanks for bearing with me

**Pull request merged**